### PR TITLE
Macgeargear/sucu 37 component tab capsule

### PR DESCRIPTION
**Pull request by macgeargear**
### Type of Change

- [x] New features
- [ ] Breaking changes
- [ ] Refactor
- [ ] Bug fixes
- [ ] Documentation updates
- [ ] Other (please specify):

## Description

- Tab capsule component
- Change text size for button
    - sm: `text-xs` (12px)
    - default: `text-base` (16px)
    - lg: `text-xl` (20px)

## Screenshots or GIFs (if applicable)
MOV Converter Screen Recording

### CODE REVIEW

File: src/lib/components/TabCapsule.svelte [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+	import { cn } from '$lib/utils';
+	import { faPlus } from '@fortawesome/free-solid-svg-icons';
+	import Fa from 'svelte-fa';
+
+	export let active: boolean = false;
+	export let label: string;
+
+	let isHovered = false;
```
Comment (punchanabu) [inline src/lib/components/TabCapsule.svelte]: I think the component doesn't have `hovered` state can you remove this kub ?

File: src/lib/components/TabCapsule.svelte [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+<button
+	class={cn(
+		'gap-2 flex items-center rounded-full w-fit px-2 border border-sucu-gray-light text-sucu-gray-light cursor-pointer hover:border-sucu-pink-01 hover:text-sucu-pink-01',
+		{
+			'border-sucu-pink-01 text-sucu-pink-01': active
+		}
+	)}
+	on:mouseenter={() => (isHovered = true)}
+	on:mouseleave={() => (isHovered = false)}
```
Comment (punchanabu) [inline src/lib/components/TabCapsule.svelte]: then can you remove  this props too ? the component doesn't have the `hovered` state

File: src/lib/components/TabCapsule.svelte [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	on:click
+>
+	<span class="text-sm md:text-base">{label}</span>
+	<Fa
+		icon={faPlus}
+		class={cn('transition-transform transform', {
+			'rotate-45': isHovered || active
```
Comment (punchanabu) [inline src/lib/components/TabCapsule.svelte]: change to `'rotate-45': active`.

File: src/lib/components/TabCapsule.svelte [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+	export let active: boolean = false;
+	export let label: string;
+
+	let isHovered = false;
+</script>
+
+<button
+	class={cn(
+		'gap-2 flex items-center rounded-full w-fit px-2 border border-sucu-gray-light text-sucu-gray-light cursor-pointer hover:border-sucu-pink-01 hover:text-sucu-pink-01',
```
Comment (punchanabu) [inline src/lib/components/TabCapsule.svelte]: remove `hover:border-sucu-pink-01`, `hover:text-sucu-pink-01`

**Review by punchanabu — CHANGES_REQUESTED** [review]

File: src/lib/components/TabCapsule.svelte (hunk 1 above)
Comment (macgeargear) [reply src/lib/components/TabCapsule.svelte]: Mov to Gif conversion
do you mean this isn't required, just change state when click right?

Comment (punchanabu) [reply src/lib/components/TabCapsule.svelte]: Yep

Comment (macgeargear) [reply src/lib/components/TabCapsule.svelte]: Ok ngub

**Review by punchanabu — APPROVED** [review]
Good job Bro !

**Pull request merged**